adding rum-cli-examples (#8984)
aws/aws-cli: awscli/examples/rum/update-app-monitor.rst

**To update the configuration of an existing app monitor**

> The following ``update-app-monitor`` updates the configuration of an existing app monitor. If the command succeeds, no output is returned. ::

Reviewer: ```suggestion
The following ``update-app-monitor`` example updates the configuration of an existing app monitor. ::
```

Author: fixed line

> 	aws rum update-app-monitor \

Reviewer: Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general

		--app-monitor-configuration 'AllowCookies=true,EnableXRay=true' \
		--custom-events Status=ENABLED \
		--name AWSApp

> 

Reviewer: ```suggestion

This command produces no output.

```

Author: modified line 10
